pull_data.php

```diff
@@ -16,37 +14,55 @@ function about_output(pdo $db) :array
 $about_result=about_output($db);
 
 
+
 /*
- * takes the array $about_result and turns it into a single key value of bio
+ * takes the array $about_result and turns it into a single key value of bio and finds out if the value is null
  *
  * @$param array this is an array of fields from the about_me section of the db
  *
- * @returns string values from the key bio
+ * @returns string values from the key bio or if value is null it returns no
  */
-function output_bio(array $about_result) :string {
-    return $about_result['bio'];
+function output_bio(array $about_result) :string
+{
+    if($about_result === 0) {
+        return 'false';
+        }
+        else{
+            return $about_result['bio'];
+        }
 }
 
 /*
- * takes the array $about_result and turns it into a single key value of life_now
+ * takes the array $about_result and turns it into a single key value of life_now and finds out if the value is null
```

Comment:
The descriptions are not very descriptive. Are yo sure you mean key value?

Reply:
Same problems with the above function about return type hinting and description and I imagine fr the one below.

These functions are so similar that they should really be one function. But ya know, up to you.